notebook/base/handlers.py

```diff
@@ -293,6 +293,14 @@ def check_origin(self, origin_to_satisfy_tornado=""):
     # template rendering
     #---------------------------------------------------------------
     
+    @property
+    def nbextensions_tree(self):
+        return self.settings.get('nbextensions_tree', {})
+
+    @property
+    def nbextensions_notebook(self):
```

Comment:
jupyter_cms installs an extension for the "edit" view as well. Should that be supported throughout as well? (edit.json is currently supported.)